conf/messages.cy

```diff
@@ -106,6 +106,7 @@ ers_check_file.file_type_error =Gwnaethoch ddewis gwirio ffeil ODS, ond nid yw {
 ers_check_file.no_file_error =Dim ffeil wedi’i dewis. Dewiswch eich ffeil.
 
 ers_upload_form.processing_alert = Rydym yn gwirio eich ffeil am wallau. Peidiwch ag adnewyddu’r dudalen.
+ers_upload_form.processing_spinner_alt = Rydym yn gwirio eich ffeil am wallau...
```

Comment:
Where is the translation from? Does it actually mean: File checking in progress... ? or maybe: We’re checking your file for errors. :)

Reply:
Your're right, it's a copy of the first sentence in the alert message, so it means "we're checking your file for errors".  I can't speak Welsh so I figured this was the best we could do for alt text!